### PR TITLE
Updating github changes command in post release workflow (changelog fix)

### PR DESCRIPTION
**Pull request by KasukabeDefenceForce**
### :pencil: Description

**Type:** :beetle: `bugfix` | :rocket: `feature` | :biohazard: `breaking change` | :vertical_traffic_light: `testing` | :memo: `documentation` | :roller_coaster: `infrastructure`

Fixing the post release workflow by updating the github changes command.

Also, link issues affected by this pull request by using the keywords: `close`, `closes`, `closed`, `fix`, `fixes`, `fixed`, `resolve`, `resolves` or `resolved`. 

### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [ ] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

File: .github/workflows/post-release.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
         run: sudo npm install -g github-changes
 
       - name: Update changelog
-        run: github-changes -o tardis-sn -r tardis --only-pulls --use-commit-body -f CHANGELOG.md -k ${{ secrets.BOT_TOKEN }}
```
Comment (atharva-2001) [inline .github/workflows/post-release.yml:27]: Were you able to run the command locally?

Comment (atharva-2001) [reply .github/workflows/post-release.yml:27]: We'll need the token
https://stackoverflow.com/questions/33655700/github-api-fetch-issues-with-exceeds-rate-limit-prematurely

File: .github/workflows/post-release.yml [hunk 2]
```diff
@@ -20,11 +20,19 @@ jobs:
           token: ${{ secrets.BOT_TOKEN }}
           fetch-depth: 0
 
-      - name: Install github-changes
-        run: sudo npm install -g github-changes
+      - name: Check out repository
```
Comment (atharva-2001) [inline .github/workflows/post-release.yml:23]: did this twice 😄

Comment (KasukabeDefenceForce) [reply .github/workflows/post-release.yml]: Fixed.

**Comment by andrewfullard** [comment]
Can you provide an example changelog entry?

**Comment by KasukabeDefenceForce** [comment]
> Can you provide an example changelog entry?

https://github.com/KasukabeDefenceForce/tardis/actions/runs/9645564434

**Comment by andrewfullard** [comment]
> > Can you provide an example changelog entry?
> 
> https://github.com/KasukabeDefenceForce/tardis/actions/runs/9645564434

Looks good, I assume the missing version and incorrect date is because it's not attached to an actual release?

**Comment by KasukabeDefenceForce** [comment]
> > > Can you provide an example changelog entry?
> > 
> > 
> > https://github.com/KasukabeDefenceForce/tardis/actions/runs/9645564434
> 
> Looks good, I assume the missing version and incorrect date is because it's not attached to an actual release?

Yes

**Review by atharva-2001 — APPROVED** [review]

**Pull request merged**